aws/services.go

```diff
@@ -0,0 +1,17 @@
+package aws
+
+// SupportedServices is a list of all AWS services that are supported by Terracost.
+var SupportedServices = []string{
+	"AmazonEC2",
+	"AmazonRDS",
+}
+
+// IsServiceSupported returns true if the AWS service is valid and supported by Terracost (e.g. for ingestion.)
+func IsServiceSupported(service string) bool {
+	for _, s := range SupportedServices {
+		if s == service {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Haha true, the list is known ahead of time and very small so I didn't care about that, especially since the main purpose of it is to expose that list to library users. So a `map[string]struct{}` and `func GetSupportedServices() []string` would be better?

Reply:
I would keep the `IsServiceSupported` as it's easy access, and add the `GetSupportedServices` to just return the keys, potentially sorted if order is something we care.